### PR TITLE
Standardized dates across site. Added confirmation windows to deleting announcements and tasks.

### LINKED ISSUE #255: Confirmation should be required before discontinuing a task
Work with backend; I'm thinking that clicking Discontinue on the Edit Task page pops up (or redirects to) something that says, "Are you sure you want to discontinue this task?" with Yes and No buttons.

### PR DESCRIPTION
**Pull request by snuriel92**
I think all the dates are the same now, I couldn't think of anywhere else we have them.

Closes #255, #254, #253

### CODE REVIEW

File: fullhouse/templates/edit_announcement.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       <input type="hidden" name="id" value="{{ id }}" />
       {{ form.as_p }}
       <input type="submit" name="save" value="{% trans 'Save Changes' %}" />
-      <input type="submit" name="delete" value="{% trans 'Delete' %}" />
+      <input type="submit" name="delete" onclick="return confirm('Are you sure you want to delete this announcement?');" value="{% trans 'Delete' %}" />
```
Comment (j3ffml) [inline fullhouse/templates/edit_announcement.html]: I'm not familiar with return confirm. Is that a popup? If so, I would suggest we use a modal dialog instead (bootstrap has js ones we can use)

**Comment by natmote** [comment]
works as advertised in chrome

@jlowdermilk, it is a popup, and I'd agree that it could be prettier.

I won't merge since we have an unanswered question.

File: fullhouse/templates/edit_announcement.html (hunk 1 above)
Comment (snuriel92) [inline fullhouse/templates/edit_announcement.html]: Ya, it's a confirm popup that continues with the action if you say yes and returns if you cancel

Comment (natmote) [inline fullhouse/templates/edit_announcement.html]: can you look in to Jeff's suggestion of using twitter bootstrap?

Comment (natmote) [inline fullhouse/templates/edit_announcement.html]: also, will this work on older browsers?

Comment (snuriel92) [inline fullhouse/templates/edit_announcement.html]: I'll try to look into it tomorrow or Friday

**Comment by natmote** [comment]
What does #267 mean for this pull request? Shahaf, will you integrate the bootstrappy stuff into here?

**Comment by snuriel92** [comment]
Ya, just ignore this pull request for a bit and I'll use Jeff's stuff to change it here. He and I talked about it today.

File: fullhouse/templates/edit_task.html [hunk 2]
```diff
@@ -48,9 +48,28 @@
...
+                $('#edit-task-form').append(delete_input).submit();
+              }
+            },
+            {
+              'label': 'Cancel',
+              'class': 'btn-primary'
+            }
+          ]);
+          return false;
+        });
```
Comment (snuriel92) [inline fullhouse/templates/edit_task.html]: @creisman check here please

File: fullhouse/templates/edit_announcement.html [hunk 3]
```diff
@@ -14,12 +14,31 @@
...
+              }
+            },
+            {
+              'label': 'Cancel',
+              'class': 'btn-primary'
+            }
+          ]);
+          return false;
+        });
+      </script>
```
Comment (snuriel92) [inline fullhouse/templates/edit_announcement.html:41]: @creisman  check here plese

Comment (creisman) [inline fullhouse/templates/edit_announcement.html:41]: I thought delete was working?

File: fullhouse/templates/edit_task.html (hunk 2 above)
Comment (creisman) [inline fullhouse/templates/edit_task.html]: I think the input needs a non-empty value field.
Try <input type='hidden' name='discontinue' value='discontinue' />

File: fullhouse/templates/edit_announcement.html (hunk 3 above)
Comment (creisman) [inline fullhouse/templates/edit_announcement.html:41]: Try <input type='hidden' name='delete' value='delete' />

Comment (snuriel92) [inline fullhouse/templates/edit_announcement.html:41]: Ya, I don't know what happened to it all of a sudden. I'll try these changes. Thanks!

**Comment by snuriel92** [comment]
All fixed now! "name" needed to be "id" at the form. I think I ctrl  z'ed on announcements earlier to check something and accidentally messed that up  :)

**Comment by creisman** [comment]
LGTM

**Pull request merged**